myapp/spec/models/user_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validaton' do
+    it 'create user successfully' do
+      user = build(:user)
+      expect(user).to be_valid
+    end
+
+    context 'when username is empty' do
```

Comment:
[FYI]
As I said "How about using context?", each person has his/her own preference and there is no one good way to write this.
Both those who nest with `context` and those who write tests all on the same hierarchy with `it` have their good points. 🙏 

I will share one reference on how to write a spec. 
https://www.betterspecs.org/